src/xdocs/config_imports.xml

```diff
@@ -818,6 +822,42 @@ import android.*;
           which is part of the Checkstyle distribution.
         </p>
 
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>
+          Short description of the behaviour:
+          <ul>
+            <li>Check starts checking from the closest to the current package described inside import control file.</li>
```

Comment:
Yes, I am not sure about "closest" and "current" here.
I assume this is trying to reference https://github.com/checkstyle/checkstyle/blob/ce21086e087661553f3a774c38362327ee88996a/src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControl.java#L195 .
Instead of `closest` why don't we go with "longest matching sub-package"?